test/plugin/scenarios/zookeeper-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,133 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {zookeeper-scenario: 2}
+  instances:
+    - {zookeeper-scenario: 1}
+  operationNames:
+    - zookeeper-scenario: [Zookeeper/WatchedEvent/NodeDeleted, /zookeeper-scenario/case/zookeeper-case,
+                           Zookeeper/WatchedEvent/None, /zookeeper-scenario/case/healthCheck]
```

Comment:
Why I cannot find the `Zookeeper/WatchedEvent/None` Segment in `SegmentItems`? 
Am I missing things?

Reply:
At begin, when `healthCheck` 
![image](https://user-images.githubusercontent.com/10150229/71342557-140eea80-2598-11ea-9bf4-5d1991e81352.png)
The delete method trigger `Zookeeper/WatchedEvent/None`,
So I do not assertion it

Reply:
I had remove this unuse SegmentItem